src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -678,8 +687,31 @@ public void Commit(bool previewChanges = false)
         /// <returns><see langword="true"/> if the rename operation was commited, <see
         /// langword="false"/> otherwise</returns>
         private bool CommitWorker(bool previewChanges)
```

Comment:
could we just remove this method and have the caller do the jtf.Run?  Would remove at least 1 layer

Reply:
the reason for this member is that the TestAccessor directly calls into this and wants to get the bool back.  I could expose the async member though from the test accessor if there isn't huge fallout from that.  will have to see about the test consumption side.  if it's already async, then i'll do that.

Reply:
made async.